src/main/java/com/griefcraft/lwc/LWC.java

```diff
@@ -793,6 +794,17 @@ public boolean canAccessProtection(Player player, Protection protection) {
                     }
                 }
 
+                break;
+            case FACTION:
```

Comment:
I know that you can use custom permissions for that, but it's not exactly what i was searching for. I don't want every member of a faction to have access to private protections. I want an option for the player to grant access to his faction's members.
The existing access event works for faction admins, as they can always access private protections, but normal members should only be allowed to do so, if the owner enabled it - or as in this PR, selected the given protections type.
If there is already a way to do so, i'm open to hear about it ^^

Reply:
I haven't played Factions in a while so maybe you can explain this point in a bit more detail -- how is "every member of a faction" different from "his faction's members". It sounds like the same group to me.

Adding specific members is already a thing, e.g. `/cmodify <username> <username2> ...`

I still believe a protection type is the wrong choice for this. If the implications are that permissions are not powerful enough for what you are attempting to add this feature for, let's discuss that further first. There is probably still some way to represent the relationship properly.

Reply:
What I meant is that a player doesn't necessarly want every faction's member to have access to his protections out of the box. But he should have the option to enable such for a specific protection. So private protections stay private, but a player can create a "faction" protection, which indeed allows the other members to gain access. 

Of course you can use `/cmodify <username>` manually for every member and protection, but this does take a lot more effort to do and doesn't even consider players leaving or joining a faction. A protection type would be a dynamic and easy way to do so.

I hope I could explain it a bit more clear :D

Reply:
Yeah so based on what you're saying it would make much more sense to add a faction to a protection's ACL than make it a protection type.

As a reminder, protection types represent the type of access members have by default. For example, public protections allow any interaction from outside members. Private protections allow no interaction from outside members. Donation protections only allow placing items inside.

On the flip side, the ACL grants actual permission to use the protection fully. For example, if you add a user to a private protection, they are able to use it whereas others cannot. If you add someone to a donation protection, they can take items, not only place.

> ... doesn't even consider players leaving or joining a faction. A protection type would be a dynamic and easy way to do so.

Sounds like this might be the source of confusion. `/cmodify` allows for much more than just usernames to be entered. See the [documentation](https://github.com/pop4959/LWCX/wiki/Commands#protection-management). Town and regions can be added to the ACL, and as I suggested already before, this is where you would want to add factions support as well.

Reply:
Ah okay, I didn't think of that option. I will try that, thank you ^^

Also, should I close this Pull request and open another one when I'm done, or should I just change this one, according to the new specifications?

Reply:
Either way is fine